src/napi/node_api_async.c

```diff
@@ -57,6 +61,8 @@ static void iotjs_uv_work_after_cb(uv_work_t* req, int status) {
       jerry_release_value(jval_err);
     }
   }
+
+  iotjs_process_next_tick();
```

Comment:
Please add a regression on https://github.com/Rokid/ShadowNode/blob/1a7a7649be26330cebde42131808142eaede5b3b/test/napi/napi_async.test.js :)

Reply:
tests are already included https://github.com/Rokid/ShadowNode/blob/1a7a7649be26330cebde42131808142eaede5b3b/test/napi/napi_async.test.js#L12.
Yet it doesn't test as expected though.